core/testing/include/Kiso_Testing.h

```diff
@@ -0,0 +1,192 @@
+/**********************************************************************************************************************
```

Comment:
We should discuss about the location of testing

Reply:
testing is a shared library between the test cases written in e.g core/essentials or boards and the test application. I think this is the place where it has to be located

Reply:
I'm unfortunately not convinced. I still believe that high cohesion is important, we need to discuss it the 3 of us.